vendor/engines/sanger_sequencing/app/models/sanger_sequencing/submission.rb

```diff
@@ -22,9 +22,15 @@ class Submission < ActiveRecord::Base
     alias purchased_at ordered_at
 
     scope :purchased, -> { joins(:order).merge(Order.purchased.order(ordered_at: :desc)) }
-    scope :ready_for_batch, -> { purchased.merge(OrderDetail.new_or_inprocess).where(batch_id: nil) }
     scope :for_facility, ->(facility) { where(orders: { facility_id: facility.id }) }
 
+    BATCHABLE_STATES = %w(new inprocess complete).freeze
```

Comment:
If they are attached to a batch, they shouldn't show up in this scope, which I believe is the way this works..? does that make sense